app/views/content/blog/a-diary-of-a-trainee-teacher.md

```diff
@@ -31,9 +31,9 @@ When lockdown hit in 2020, I began to reassess my career choices as I knew I had
 
 ## The first few days
 
-My course started on 1st September, and I spent the first three days with my School Direct partner, Sheffield Teacher Training Alliance. This was a fantastic opportunity to get to know the rest of the cohort, including students training to teach other subjects on the curriculum. Sheffield Teacher Training Alliance had organised various sessions over these three days which were linked to the [Initial Teacher Training Core Content Framework](https://www.gov.uk/government/publications/initial-teacher-training-itt-core-content-framework). 
+My course started on 1st September, and I spent the first three days with Sheffield Teacher Training Alliance. This was a fantastic opportunity to get to know the rest of the cohort, including students training to teach other subjects on the curriculum. Sheffield Teacher Training Alliance had organised various sessions over these three days which were linked to the [Initial Teacher Training Core Content Framework](https://www.gov.uk/government/publications/initial-teacher-training-itt-core-content-framework). 
 
-For some sessions, outside speakers came in to deliver the lesson. I particularly enjoyed the sessions on attention deficit hyperactivity disorder (ADHD) and dyslexia, where our presenters gave us a real insight into what it feels like to have those conditions. The activities we took part in really opened my eyes to the challenges faced and provided me with practical advice on how to support a student with ADHD and/or dyslexia. At the end of the Friday session, those of us studying with the University of Sheffield said goodbye to our School Direct cohort (for a little while anyway!) and relaxed over the weekend ready for our first week at university!
+For some sessions, outside speakers came in to deliver the lesson. I particularly enjoyed the sessions on attention deficit hyperactivity disorder (ADHD) and dyslexia, where our presenters gave us a real insight into what it feels like to have those conditions. The activities we took part in really opened my eyes to the challenges faced and provided me with practical advice on how to support a student with ADHD and/or dyslexia. At the end of the Friday session, those of us studying with the University of Sheffield said goodbye to our school-based cohort (for a little while anyway!) and relaxed over the weekend ready for our first week at university!
```

Comment:
not sure it's super clear here what 'school-based cohort' means - but also not sure there's much we can do given it's a blog (bit late to try and clarify with the writer given it's from 2021 ...)